nautobot/extras/views.py

```diff
@@ -1107,8 +1107,9 @@ def get_extra_context(self, request, instance):
         elif related_object:
             associated_record = related_object
 
-        logs = JobLogEntry.objects.restrict(request.user, "view").filter(job_result=instance)
-        log_table = tables.JobLogEntryTable(data=logs, user=request.user)
+        # This is used as a sentinel for backwards compatibility since the table
+        # object is rendered inside of `JobLogEntryTableView`.
+        log_table = True
```

Comment:
Stilil want to revisit this with the SSoT plugin to make sure we still need this.

Reply:
Ref: https://github.com/nautobot/nautobot-plugin-ssot-servicenow/pull/1